docs/development.md

```diff
@@ -54,9 +54,14 @@ vagrant up centos7-devel
 The box can now be accessed via ssh and the Rails server started directly (this assumes you are connecting as the default `vagrant` user):
 
 ```sh
+cd forklift
+vagrant ssh centos7-devel -- -L3000:localhost:3000
 vagrant ssh centos7-devel
```

Comment:
remove this line

Reply:
No issues with the removal of this line however, would be good to have it mentioned as an optional command in case user want to map to the local port..